plugins/modules/fabric_devices_workflow_manager.py

```diff
@@ -0,0 +1,4080 @@
+                    - If layer3_handoff_ip_transit is set, border type will be considered as Layer3.
+                    type: list
+                    elements: dict
+                    suboptions:
+                      transit_network_name:
+                        description:
+                        - Network that connects multiple SDA fabrics or networks.
+                        - Mandatory parameter for all operations in L3 Handoff with IP transit.
+                        - transit_network_name cannot be updated.
+                        type: str
+                      interface_name:
+                        description:
+                        - Refers to the specific network interface in the border device.
+                        - Mandatory parameter for all operations in L3 Handoff with IP transit.
+                        - interface_name cannot be updated.
+                        type: str
+                      external_connectivity_ip_pool_name:
+                        description:
+                        - IP address range allocated for communication between the SDA fabric and external networks.
+                        - Mandatory parameter for adding the L3 Handoff with IP transit.
+                        - This IP pool should be reserved in the fabric site.
+                        - If external_connectivity_ip_pool_name is set, then no need to set the local and remote addresses.
+                        - Setting external_connectivity_ip_pool_name will automatically set the local and remote adresses.
+                        - If both are set, external_connectivity_ip_pool_name will be given higher priority.
+                        - Updating the IP addresses are not permitted.
+                        type: str
+                      virtual_network_name:
+                        description:
+                        - Refers to the logical segmentation of the network, grouping devices into isolated virtual networks.
+                        - Either virtual_network_name or vlan_id is mandatory for all operations in L3 Handoff with IP transit.
+                        type: str
+                      vlan_id:
+                        description:
+                        - Unique identifier assigned to a Virtual Local Area Network
+                        - Should be unique across the entire fabric site settings.
+                        - vlan_id can be from 1 to 4094. Except 1, 1002-1005, 2046, 4094.
+                        - Either virtual_network_name or vlan_id is mandatory for all operations in L3 Handoff with IP transit.
+                        - vlan_id cannot be updated.
+                        type: int
+                      tcp_mss_adjustment:
+                        description:
+                        - Allows the modification of the Maximum Segment Size in TCP packets.
+                        - tcp_mss_adjustment can be from 500 to 1440.
```

Comment:
- The 'tcp_mss_adjustment' can be set from 500 to 1440.
                    - This parameter can be updated after being initially set.